GVFS/GVFS.Common/ServerGVFSConfig.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace GVFS.Common
 {
-    public class GVFSConfig
+    public class ServerGVFSConfig

```

Comment:
This would be easier to review if this rename was done in a separate commit. Can you tease those apart?

Reply:
Moved the rename of GVFSConfig to a separate commit.